bfdd/bfdd_nb_config.c

```diff
@@ -680,8 +879,10 @@ int bfdd_bfd_sessions_single_hop_required_receive_interval_modify(
 
 	case NB_EV_APPLY:
 		bs = nb_running_get_entry(args->dnode, NULL, true);
-		bs->peer_profile.min_rx =
-			yang_dnode_get_uint32(args->dnode, NULL);
+		if (rx_interval == bs->timers.required_min_rx)
+			return NB_OK;
+
+		bs->peer_profile.min_rx = rx_interval;
 		bfd_session_apply(bs);
```

Comment:
ditto: separate commit for changes related to common bfd.

Reply:
you seem to have removed those changes.
if the behavior can be kept unchanged, it is ok to me.